fabtests/component/sock_test.c

```diff
@@ -451,9 +461,11 @@ static int run_server(void)
 	if (ret)
 		return ret;
 
+#if HAVE_EPOLL == 1
 	ret = test_epoll();
 	if (ret)
 		return ret;
+#endif
```

Comment:
ifdefs sprinkled within functions makes maintenance difficult.  Can you add this at line 430:
```
#else
static int test_epoll()
{
    return 0;
}
```
and remove the check here?  You can also return -ENOSYS instead and update the error check after test_epoll().

Reply:
Done.